*** Apache2_modoki.html	2014-11-17 16:48:04.776192109 +0900
--- temp_Tumblr_my_Layout/Apache2_modoki_without_js.html	2014-11-17 16:54:35.104347140 +0900
***************
*** 1,5 ****
  <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
! <!-- Apache2のファイル一覧ぽくするやつ -->
  
  <html>
      <head>
--- 1,5 ----
  <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
! <!-- Apache2のファイル一覧ぽくするやつ (Without Java Script) -->
  
  <html>
      <head>
***************
*** 13,24 ****
              {/block:IfHideTumblrControls}
          </style>
  
-         <script type="text/javascript">
-             _ScrapingURL = function(url) {
-                     return url.match(".+/(.+?)([\?#;].*)?$")[1];
-             };
-         </script>
- 
      </head>
      <body>
          {block:IndexPage}
--- 13,18 ----
***************
*** 26,39 ****
              <h1>Index of {Title}</h1>
          {/block:IndexPage}
          {block:PermalinkPage}
!             <script type="text/javascript">
!                 document.title = ('Index of ' + '{Title}' + _ScrapingURL('{Permalink}') + '/');
!             </script>
!             <h1>
!                 <script type="text/javascript">
!                     document.write('Index of {Title}' + _ScrapingURL('{Permalink}') + '/');
!                 </script>
!             </h1>
          {/block:PermalinkPage}
  
          <table>
--- 20,27 ----
              <h1>Index of {Title}</h1>
          {/block:IndexPage}
          {block:PermalinkPage}
!             <title>Index of {Title}{PostID}</title>
!             <h1>Index of {Title}{PostID}</h1>
          {/block:PermalinkPage}
  
          <table>
***************
*** 63,76 ****
                              </td>
                              <td>
  
!                                 <!-- Start "Getting Filename from URL." -->
!                                 <a href={PhotoURL-HighRes}>
!                                     <script type="text/javascript">
!                                         document.write(_ScrapingURL('{PhotoURL-HighRes}'));
!                                     </script>
!                                 </a>
!                                 <!-- End "Getting Filename from URL." -->
! 
                              </td>
                              <td align="right">
                                  {DayOfMonthWithZero}-{ShortMonth}-{Year} {24HourWithZero}:{Minutes}
--- 51,60 ----
                              </td>
                              <td>
  
!                                 <!-- Without Javascript (View Long URL) -->
!                                 <a href={PhotoURL-HighRes}>{PhotoURL-HighRes}</a>
!                                 <!-- Without Javascript (View Long URL) -->
!                                 
                              </td>
                              <td align="right">
                                  {DayOfMonthWithZero}-{ShortMonth}-{Year} {24HourWithZero}:{Minutes}
***************
*** 99,111 ****
                                  </td>
                                  <td>
  
!                                     <!-- Start "Getting Posts Title (Photoset) from URL." -->
!                                     <a href={Permalink}>
!                                         <script type="text/javascript">
!                                             document.write(_ScrapingURL('{Permalink}') + '/');
!                                         </script>
!                                     </a>
!                                     <!-- End "Getting Posts Title (Photoset) from URL." -->
  
                                  </td>
                                  <td align="right">
--- 83,91 ----
                                  </td>
                                  <td>
  
!                                     <!-- Without Javascript (View Long URL) -->
!                                     <a href={Permalink}>{Permalink}</a>
!                                     <!-- Without javascript (View Long URL) -->
  
                                  </td>
                                  <td align="right">
***************
*** 135,147 ****
                                      </td>
                                      <td>
  
!                                         <!-- Start "Getting Filename from URL." -->
!                                         <a href={PhotoURL-HighRes}>
!                                             <script type="text/javascript">
!                                                 document.write(_ScrapingURL('{PhotoURL-HighRes}'));
!                                             </script>
!                                         </a>
!                                         <!-- End "Getting Filename from URL." -->
  
                                      </td>
                                      <td align="right">
--- 115,123 ----
                                      </td>
                                      <td>
  
!                                         <!-- Without Javascript (View Long URL) -->
!                                         <a href={PhotoURL-HighRes}>{PhotoURL-HighRes}</a>
!                                         <!-- Without Javascript (View Long URL) -->
  
                                      </td>
                                      <td align="right">
***************
*** 152,158 ****
                                      </td>
                                      <td align="right">
                                          {block:Caption}
!                                             {Caption}
                                          {/block:Caption}
                                      </td>
                                      <td>
--- 128,134 ----
                                      </td>
                                      <td align="right">
                                          {block:Caption}
!                                             {PhotoAlt}
                                          {/block:Caption}
                                      </td>
                                      <td>
***************
*** 169,191 ****
              <tr><th colspan="5"><hr></th></tr>
          </table>
  
!         <address>
!             {block:IfBottomStrings}
!                 {text:Bottom Strings}
!             {/block:IfBottomStrings}
!             {block:IfNotBottomStrings}
!                 Tumblr/Apache2_modoki at tumblr.com Port 80
!             {/block:IfNotBottomStrings}
!         </address>
  
          {block:PreviousPage}
!             <a href={PreviousPage}>Prev</a>
          {/block:PreviousPage}
          {block:Pagination}
!             &nbsp;{CurrentPage}/{TotalPages}&nbsp;
          {/block:Pagination}
          {block:NextPage}
!             <a href={NextPage}>Next</a>
          {/block:NextPage}
  
      </body>
--- 145,160 ----
              <tr><th colspan="5"><hr></th></tr>
          </table>
  
!         <address>Tumblr/Apache2_modoki at tumblr.com Port 80</address>
  
          {block:PreviousPage}
!             <a href={PreviousPage}>Prev </a>
          {/block:PreviousPage}
          {block:Pagination}
!             {CurrentPage}/{TotalPages}
          {/block:Pagination}
          {block:NextPage}
!             <a href={NextPage}> Next</a>
          {/block:NextPage}
  
      </body>
