*** Apache2_modoki.html	2014-11-07 05:29:17.532336996 +0900
--- Apache2_modoki_without_js.html	2014-11-07 05:28:22.188959814 +0900
***************
*** 51,65 ****
                                  </td>
                                  <td>
  
!                                     <!-- Start "Getting Filename from URL." -->
!                                     <a href={PhotoURL-HighRes}>
!                                         <script type="text/javascript">
!                                             var url = "{PhotoURL-HighRes}";
!                                             var filename = url.match(".+/(.+?)([\?#;].*)?$")[1];
!                                             document.write(filename);
!                                         </script>
!                                     </a>
!                                     <!-- End "Getting Filename from URL." -->
  
                                  </td>
                                  <td align="right">
--- 51,59 ----
                                  </td>
                                  <td>
  
!                                     <!-- Without Javascript (View Long URL)  -->
!                                     <a href={PhotoURL-HighRes}>{PhotoURL-HighRes}</a>
!                                     <!-- Without Javascript (View Long URL) -->
  
                                  </td>
                                  <td align="right">
***************
*** 88,102 ****
                                  </td>
                                  <td>
  
!                                     <!-- Start "Getting Posts Title (Photoset) from URL." -->
!                                     <a href={Permalink}>
!                                         <script type="text/javascript">
!                                             var url = "{Permalink}";
!                                             var filename = url.match(".+/(.+?)([\?#;].*)?$")[1];
!                                             document.write(filename + '/');
!                                         </script>
!                                     </a>
!                                     <!-- End "Getting Posts Title (Photoset) from URL." -->
  
                                  </td>
                                  <td align="right">
--- 82,90 ----
                                  </td>
                                  <td>
  
!                                     <!-- Without Javascript (View Long URL) -->
!                                     <a href={Permalink}>{Permalink}</a>
!                                     <!-- Without Javascript (View Long URL) -->
  
                                  </td>
                                  <td align="right">
***************
*** 129,141 ****
                                      <td>
  
                                          <!-- Start "Getting Filename from URL." -->
!                                         <a href={PhotoURL-HighRes}>
!                                             <script type="text/javascript">
!                                                 var url = "{PhotoURL-HighRes}";
!                                                 var filename = url.match(".+/(.+?)([\?#;].*)?$")[1];
!                                                 document.write(filename);
!                                             </script>
!                                         </a>
                                          <!-- End "Getting Filename from URL." -->
  
                                      </td>
--- 117,123 ----
                                      <td>
  
                                          <!-- Start "Getting Filename from URL." -->
!                                         <a href={PhotoURL-HighRes}>{PhotoURL-HighRes}</a>
                                          <!-- End "Getting Filename from URL." -->
  
                                      </td>
